Doc/library/exceptions.rst

```diff
@@ -242,6 +242,13 @@ The following exceptions are the exceptions that are usually raised.
    unqualified names.  The associated value is an error message that includes the
    name that could not be found.
 
+   The :attr:`name` attribute can be set using a keyword-only argument to the
+   constructor. When set it represent the name of the variable that was attempted
+   to be accessed.
+
+   .. versionchanged:: 3.10
```

Comment:
```suggestion
   .. versionadded:: 3.10
```

Reply:
Normally we use `versionchanged` here as the exception existed previously but it was changed (at least that's how is indicated in this document).